simulate-event.js

```diff
@@ -272,6 +292,9 @@ module.exports = function (element, type, options) {
     cancelable: true
   }, result(eventOptions, eventType, element, type, options), options);
 
+  // Preserve the original or use the new options if none were given.
+  if (!original) original = options;
```

Comment:
I don't think it's blowing any away, some properties probably aren't enumerable. This is from the change to make it accept a previously created event, right?